packages/manager/apps/zimbra/src/pages/onboarding/index.tsx

```diff
@@ -1,43 +1,76 @@
-import React from 'react';
+import React, { useContext, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { OnboardingLayout } from '@ovh-ux/manager-react-components';
-import {
-  ButtonType,
-  PageLocation,
-  useOvhTracking,
-} from '@ovh-ux/manager-react-shell-client';
+import { Card, OnboardingLayout } from '@ovh-ux/manager-react-components';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
 import onboardingImgSrc from './onboarding-img.png';
-import { JOIN_ZIMBRA_BETA } from '@/tracking.constant';
+import {
+  ZIMBRA_ADMINISTRATOR_GUIDE,
+  ZIMBRA_HOW_TO_CONFIGURE_GUIDE,
+  ZIMBRA_WEBMAIL_GUIDE,
+} from '@/guides.constants';
 
 export default function Onboarding() {
   const { t } = useTranslation('onboarding');
-
-  const title: string = t('title');
-  const description: string = t('description');
-  const { trackClick } = useOvhTracking();
+  const context = useContext(ShellContext);
+  const { ovhSubsidiary } = context.environment.getUser();
 
   const onOrderButtonClick = () => {
-    trackClick({
```

Comment:
why are we removing the tracking on this button ? I think it should be kept and the value updated if needed